pkg/asset/manifests/tectonic.go

```diff
@@ -74,6 +119,7 @@ func (t *Tectonic) Generate(dependencies asset.Parents) error {
 	}
 
 	assetData := map[string][]byte{
+		"99_cloud-creds-secret.yaml":                             applyTemplateData(content.CloudCredsSecret, templateData),
```

Comment:
We need a file for the cloud-creds-secret-reader role, too. Are you planning to add that back in with cloud-creds secret?